src/api/Areas/Dashboard/Controllers/OperatingSystemItemController.cs

```diff
@@ -98,11 +99,53 @@ public IActionResult Find()
             var user = _authorization.GetUser();
             if (user == null) return Forbid();
 
-            var result = _service.FindForUser(user.Id, filter.GeneratePredicate(), filter.Sort);
+            var result = _service.FindForUser(user.Id, filter);
             return new JsonResult(result.Select(fsi => new OperatingSystemItemModel(fsi)));
         }
     }
 
+    /// <summary>
+    /// Find all operating system items for the specified 'filter'.
+    /// Return the simple details for each operating system item.
+    /// </summary>
+    /// <returns></returns>
+    [HttpGet("list", Name = "GetOperatingSystemItemLists-Dashboard")]
+    [Produces(MediaTypeNames.Application.Json)]
+    [ProducesResponseType(typeof(IEnumerable<OperatingSystemItemListModel>), (int)HttpStatusCode.OK)]
+    [SwaggerOperation(Tags = new[] { "Operating System Item" })]
+    [ResponseCache(VaryByQueryKeys = new[] { "*" }, Location = ResponseCacheLocation.Client, Duration = 60)]
+    public IActionResult FindList()
```

Comment:
The new list endpoints reduce the amount of serialized data.  We don't need it currently, so removing it will improve performance.